File: java/1010.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in); 
        
        int cod1 = in.nextInt();
        int n1 = in.nextInt();
        double v1 = in.nextDouble();

        int cod2 = in.nextInt();
        int n2 = in.nextInt();
        double v2 = in.nextDouble();

        double total = n1 * v1 + n2 * v2;
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
    }
}

File: kotlin/1010.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);
    
    val cod1: Int = sc.next().toInt();
    val n1: Int = sc.next().toInt();
    val v1: Double = sc.next().toDouble();

    val cod2: Int = sc.next().toInt();
    val n2: Int = sc.next().toInt();
    val v2: Double = sc.next().toDouble();

    val total = n1 * v1 + n2 * v2;
    println("VALOR A PAGAR: R$ ${total.format(2)}");
}

fun Double.format(digits: Int) = "%.${digits}f".format(this)